jbehave-support-core/src/main/java/org/jbehavesupport/core/ssh/SshSteps.java

```diff
@@ -53,24 +57,38 @@
     private int maxSoftAssertCount;
 
     private ZonedDateTime scenarioStart;
+    private ZonedDateTime logReadStartTime = null;
+    private ZonedDateTime logReadEndTime = null;
+
+    private MultiKeyMap<MultiKey, String> logCache = MultiKeyMap.multiKeyMap(new LRUMap());
 
     @BeforeScenario
     public void init() {
         scenarioStart = ZonedDateTime.now();
     }
 
+    //Backwards compatibility
+    @Deprecated
```

Comment:
Since its private we can probably remove this one

Reply:
removed